ephios/core/services/notifications/types.py

```diff
@@ -333,15 +333,21 @@ def send(cls, event: Event, content: str):
                 Notification(
                     slug=cls.slug,
                     user=user,
-                    data=dict(email=participant.email, event_id=event.id, content=content),
+                    data=dict(
+                        email=participant.email,
+                        event_id=event.id,
```

Comment:
Do we need the id, still?

Reply:
not at the moment, but I think we should keep the ability to fetch the event from the db for sending to query for more information (if it exists)